src/cli/facade/dev.js

```diff
@@ -43,7 +44,9 @@ module.exports = function(dependencies) {
           if (!hotReloading) {
             logger.warn(strings.messages.cli.HOT_RELOADING_DISABLED);
           } else {
-            cb(components);
+            cb(components, done => {
+              liveReloadServer.refresh('/');
```

Comment:
how do you pick this url? `/`?

Reply:
The refresh method need you pass a string for the file that was changed. In our case for the moment that's not relevant so I'm passing some dummy content `/` to simply refresh the listening view (in our case the `preview`). 

(In the future we co-ordinate livereload to work even more closely with out browser-client so that we could perform some sort of hot-reloading. For example immagine using this with the chrome-extention tool to just re-render that specific component...)